packages/contracts-bedrock/contracts/universal/OptimismMintableERC20Factory.sol

```diff
@@ -32,13 +32,13 @@ contract OptimismMintableERC20Factory {
     /**
      * @notice Emitted whenever a new OptimismMintableERC20 is created.
      *
-     * @param localToken  Address of the created token on the local chain.
      * @param remoteToken Address of the corresponding token on the remote chain.
+     * @param localToken  Address of the created token on the local chain.
      * @param deployer    Address of the account that deployed the token.
      */
     event OptimismMintableERC20Created(
-        address indexed localToken,
         address indexed remoteToken,
+        address indexed localToken,
```

Comment:
Is it better to change the ordering here in the definition, or where it's emitted?

https://github.com/ethereum-optimism/optimism/blob/761ab09335b0a1496d05a6e9d4ea9c41432198e1/packages/contracts-bedrock/contracts/universal/OptimismMintableERC20Factory.sol#L99

Could this possible cause issues for existing tools Quixotic has written?

Reply:
Coming back to this, I see that the ordering in the `StandardBridge` matches the current definition rather than this change, and I think that we should standardize on `local` then `remote` ordering. cc @sam-goldman as we should make sure the [NFT events](https://github.com/ethereum-optimism/optimism/blob/2b273136f7e3541b246f35e38d009d1336862844/packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721Factory.sol#L21) match that.

https://github.com/ethereum-optimism/optimism/blob/9bb9fb2835551fbf4771b4489be2bdf3a271de0b/packages/contracts-bedrock/contracts/universal/StandardBridge.sol#L111-L137

Reply:
Fixed this [here](https://github.com/ethereum-optimism/optimism/pull/3491)